portfolio/src/main/webapp/style.css

```diff
@@ -1,65 +1,69 @@
 .background {
   background-color: #ff9999;
+  font: 20px 'Georgia';
 }
-
+ 
```

Comment:
Looks like there are some trailing whitespaces added in this file. Please remove them.

https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace

Reply:
Got it! Thanks for catching that :)